hw/ip/alert_handler/data/alert_handler.hjson

```diff
@@ -107,31 +107,37 @@
       local: "true"
     },
     { name: "LOCAL_ALERT_ID_ALERT_PINGFAIL",
-      desc: "Local alert ID",
+      desc: "Local alert ID for alert ping failure.",
       type: "int",
       default: "0",
       local: "true"
     },
     { name: "LOCAL_ALERT_ID_ESC_PINGFAIL",
-      desc: "Local alert ID",
+      desc: "Local alert ID for escalation ping failure.",
       type: "int",
       default: "1",
       local: "true"
     },
     { name: "LOCAL_ALERT_ID_ALERT_INTEGFAIL",
-      desc: "Local alert ID",
+      desc: "Local alert ID for alert integrity failure.",
       type: "int",
       default: "2",
       local: "true"
     },
     { name: "LOCAL_ALERT_ID_ESC_INTEGFAIL",
-      desc: "Local alert ID",
+      desc: "Local alert ID for escalation integrity failure.",
       type: "int",
       default: "3",
       local: "true"
     },
     { name: "LOCAL_ALERT_ID_BUS_INTEGFAIL",
-      desc: "Local alert ID",
+      desc: "Local alert ID for bus integrity failure.",
+      type: "int",
+      default: "4",
+      local: "true"
+    },
+    { name: "LOCAL_ALERT_ID_LAST",
+      desc: "Last local alert ID.",
```

Comment:
Is this default 4 or 5?

Reply:
And is there any information/description about the alert_id_last?

Reply:
I believe we've added this for SW convenience. In this particular case the last local alert ID corresponds to `LOCAL_ALERT_ID_BUS_INTEGFAIL` (which is ID number 4), but if we had to add more local alerts this would have to increase.

Reply:
Actually, you are on to something... we've already added more local alerts due to the shadow regs - hence we have to update this. Let me do that.

Reply:
I see, that means to be the same as the local_alert_id_bus_integfail :) Thanks for explaining.